packages/bff/src/services/catalogService.ts

```diff
@@ -122,5 +123,41 @@ export function catalogServiceBuilder(
 
       return response;
     },
+    createEService: async (
+      eServiceSeed: bffApi.EServiceSeed,
+      { headers }: WithLogger<BffAppContext>
+    ): Promise<CreatedResource> => {
+      const { id } = await catalogProcessClient.createEService(eServiceSeed, {
+        headers,
+      });
+      return { id };
+    },
+    updateEServiceById: async (
+      eServiceId: string,
+      updateEServiceSeed: bffApi.UpdateEServiceSeed,
+      { headers }: WithLogger<BffAppContext>
+    ): Promise<CreatedResource> => {
+      const { id } = await catalogProcessClient.updateEServiceById(
+        updateEServiceSeed,
+        {
+          headers,
+          params: {
+            eServiceId,
+          },
+        }
+      );
+      return { id };
+    },
+    deleteEService: async (
+      eServiceId: string,
```

Comment:
[Very minor]
I would like to uniform to naming convention and if possibile introduce Branded type with `EServiceId`, anyway params require `eserviceId`, anyway I suggest to using always the same naming convention, but it's not strictly necessary.
```suggestion
      eserviceId: EServiceId,
```

Reply:
In this case the param accept `eServiceId` and this why I call it that way, I added the branded type 👍